Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -853,26 +853,54 @@ script:
       type: Date
   - arguments:
     - default: true
-      description: The query to run.
+      description: The query to run. Must be passed if query_batch argument is empty.
       isArray: false
       name: query
-      required: true
+      required: false
       secret: false
     - default: false
       defaultValue: '10'
-      description: The amount of time (in seconds) that a request waits for the query response before a timeout occurs.
+      description: The amount of time (in seconds) that a request waits for the query response before a timeout occurs. If specified with query_batch, will be applied to all queries in the array.
       isArray: false
       name: timeout
       required: false
       secret: false
     - name: time_range
-      description: Time range to look back. Expected syntax is a human readable time range, e.g. 60 minutes, 6 hours, 1 day, etc.
+      description: Time range to look back. Expected syntax is a human readable time range, e.g. 60 minutes, 6 hours, 1 day, etc. If specified with query_batch, will be applied to all queries in the array.
       required: false
       isArray: false
       defaultValue: ''
       predefined:
       - ''
-    deprecated: false
+    - default: false
+      description:  |-
+        A JSON array of queries, limited to 10 queries. Must be passed if query argument is empty. Example for input:
+        { "queries": [
+            {
+            "query": "query #1",	// Mandatory
```

Comment:
let's remove the comments here and describe below which field is mandatory or not.